substrate-node/support/src/resources.rs

```diff
@@ -80,30 +80,31 @@ impl Resources {
         let su = self.get_su();
         cu * 2 + su
     }
+}
 
-    pub fn has_changed(&self, other: Resources, tolerance: u8) -> Result<(), ()> {
-        let wiggle = |a: u64, b: u64| -> Result<(), ()> {
-            let p = Percent::from_percent(tolerance) * a;
-            let diff = (a as i64 - b as i64).abs() as u64;
-            if diff > p {
-                return Err(());
-            }
-            Ok(())
-        };
-
-        wiggle(self.cru, other.cru)?;
-        wiggle(self.sru, other.sru)?;
-        wiggle(self.hru, other.hru)?;
-        wiggle(self.mru, other.mru)?;
+pub fn has_changed(
```

Comment:
function naming could be more explicit:
- rename to `resource_has_changed()`
- or keep function `impl Resources` as before

Reply:
I would make it static indeed.